src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,85 @@
+import classNames from 'classnames';
+import { FilterStatus } from '../types/Todo';
+
+type Props = {
+  filterStatus: string;
+  setFilterStatus: (status: FilterStatus) => void;
+  todosLenght: number;
+  isClearButtonDisabled: boolean;
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  filterStatus,
+  setFilterStatus,
+  todosLenght,
+  isClearButtonDisabled,
+}) => {
+  const clickHandlerCompleted = () => {
+    setFilterStatus(FilterStatus.COMPLETED);
+  };
+
+  const clickHandlerAll = () => {
+    setFilterStatus(FilterStatus.ALL);
+  };
+
+  const clickHandlerActive = () => {
+    setFilterStatus(FilterStatus.ACTIVE);
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {todosLenght} items left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
```

Comment:
You could use 'map' to create filter buttons so that you won't need 3 different handlers (Object.keys(YourEnum).map ... )—something like that.

Reply:
Good point